donkstation/code/datums/diseases/advance/symptoms/rodman.dm

```diff
@@ -0,0 +1,115 @@
+/datum/symptom/rodman_syndrome
+	name = "Rodman Syndrome"
+	desc = "This virus causes rapid growth of previously dormant neural connections, causing a large urge to grunt in a primal way. It is believed that this is an evolution of the failed experimental ALZ-112 virus."
+	stealth = -2
+	resistance = -2
+	stage_speed = 2
+	transmission = -4 	//Prevents airborne spread, on-contact is the highest possible
+	level = 0 			//Base symptom is very 'clown tier'
+	severity = 1 		//Non-harmful, but annoying
+	symptom_delay_min = 8
+	symptom_delay_max = 10
+	var/restore_mind = FALSE
+	var/increase_volume = FALSE
+	var/awaken_monkey = FALSE
+	var/is_monkey = FALSE
+	threshold_desc = "<b>Resistance 10:</b> The symptom now causes restoration of neural tissue in more traditional areas of the brain.<br>\
+					<b>Transmission 8:</b> The symptom grows in intensity, causing the afflicted to begin grunting and screeching more often and with higher volume."
+
+/datum/symptom/rodman_syndrome/severityset(datum/disease/advance/A)
+	. = ..()
+	if(A.transmission >= 8)
+		severity += 1 //Becomes MORE annoying, still 'harmless'
+	if(A.resistance >= 10)
+		severity -= 2 //Becomes a healing symptom at this point.
+	if((A.stealth >= 2) && (A.stage_rate >= 12))
```

Comment:
not mentioned in threshold desc

Reply:
"The symptom now causes restoration of neural tissue in more traditional areas of the brain."

Unless you mean the hidden threshold there, then that is entirely deliberate.
The zombie threshold for Autophagocytosis Necrosis is also hidden, this is meant to be a harmless counterpart to that symptom.